### PR TITLE
Register DASKR.jl

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
DASKR.jl is a wrapper for the DASKR FORTRAN DAE solver. it has a simple interface and an interface which matches JuliaDiffEq.

### CODE REVIEW

**Comment by tkelman** [comment]
Don't use Pkg.dir, use `dirname(@__FILE__)`

Should acknowledge the daskr license in the top level license file.

**Comment by tkelman** [comment]
If you use Compat, it needs to be listed in REQUIRE

**Comment by ChrisRackauckas** [comment]
Put in a new patch v0.0.2 which makes those changes.

**Comment by tkelman** [comment]
@aviks please leave new package registrations open at least 24 hours

**Comment by aviks** [comment]
yes, sorry, i misread the date.

File: DASKR/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+DiffEqBase
+Compat
```
Comment (ChrisRackauckas) [inline DASKR/versions/0.0.2/requires]: Yes, future patch will have it.

**Pull request merged**